pycrdt_websocket/yroom.py

```diff
@@ -229,7 +232,7 @@ async def serve(self, websocket: Websocket):
                             )
                             tg.start_soon(client.send, message)
             except Exception as e:
-                self.log.debug("Error serving endpoint: %s", websocket.path, exc_info=e)
+                self.log.error("Error serving endpoint: %s", websocket.path, exc_info=e)
```

Comment:
Should we remove a client if we cannot serve them?
```suggestion
                self.log.error("Error serving endpoint: %s", websocket.path, exc_info=e)
                self.clients.remove(websocket)
```